repositories.bzl

```diff
@@ -1,18 +1,21 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 def com_google_protoc_java_resource_names_plugin_repositories(
+    omit_pypi_six = False,
     omit_com_google_protobuf = False,
     omit_pypi_py_yaml = False,
     omit_pypi_pystache = False,
-    omit_pypi_six = False):
+    omit_pypi_ply = False):
+    if not omit_pypi_six:
```

Comment:
Does the reordering matter here?

Reply:
No, reordering did not matter, I  just made it to group dependencies, which are imported in same way. What matters it that I forgot to add ply dependency in bazel and was importing pystache incorrectly (the `strip_prefix` argument was stripping too much).  With this is should be possible to run the generation on any machine with python installed (clean, all deps will be pulled by bazel). It still does not make the build hermetic 100%, but I think good enough for now and we can switch to pulling and building python itself in the future to achive 100% hermetic build.

Reply:
Ah ok, thanks!